python/friesian/example/recall/README.md

```diff
@@ -1,6 +1,100 @@
-https://github.com/facebookresearch/faiss/blob/main/INSTALL.md
+# Recall items by using faiss
+This example demonstrates how to use BigDL Friesian 
+to get the recalled items by using the retrieval algorithms provided by faiss.
 
-Need to install via conda to get reasonable performance on CPU:
+## Prepare the environment
+We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster (yarn-client mode only).
+Also Need to install faiss via conda to get reasonable performance on CPU, referring [faiss install](https://github.com/facebookresearch/faiss/blob/main/INSTALL.md).
```

Comment:
Also Need -> Faiss needs to be installed via conda
reasonable -> promising
referring -> . Please refer to ... for more installation guidance.

Reply:
ok~